bioagents/tra/tra.py

```diff
@@ -196,7 +191,7 @@ def plot_results(self, results, agent, obs_name, thresh=50):
         for tspan, yobs in results:
             plt.plot(tspan, yobs[obs_name])
         plt.ylim(-5, max_val_lim)
-        plt.xlim(-100, 10100)
+        plt.xlim(min_time, max_time)
```

Comment:
I think at least the minimum value of `-100` here was on purpose to add a little bit of a gap on the left hand side of the plot, making some trajectories (e.g., ones that go up quickly at the beginning of the simulation) and labels better visible. Is there a situation where `min([result[0][0] for result in results])` would be expected to be something other than 0? If not, then we could keep the fixed `-100` setting or make it adaptive with respect to `max_time`, for instance, by setting it to `-max_time/100`.

Reply:
I have an example where the time only ranges between 0 and 3 - having -100 doesn't look good so I changed the range to use `-max_time/100, max_time+max_time/100`

Reply:
Thanks!